README.md

```diff
@@ -124,8 +124,12 @@ The image exposes ports `80` and `3306`, and exports four volumes:
 * `/var/log/httpd`, containing Apache log files.
 * `/var/log/mysql` containing MariaDB log files.
 * `/var/www/html`, used as Apache's [DocumentRoot directory](http://httpd.apache.org/docs/2.4/en/mod/core.html#documentroot).
-* `/var/lib/mysql`, where MariaDB data files are stores.
+* `/var/lib/mysql`, where MariaDB data files are stored.
+* `/etc/apache2`, where Apache configuration files are stored.
```

Comment:
I think we can just add "/etc/apache2, where Apache configuration files are stored. Please, refer to https://docs.docker.com/storage/volumes for more information on using host volumes." instead.

Reply:
Fine by me, that's better than not having it at all.